src/cork.rs

```diff
@@ -62,36 +74,20 @@ impl proto::contract_call_service_server::ContractCallService for CorkHandler {
 
         debug!("hex encoded call: {:?}", hex::encode(&encoded_call));
 
-        match schedule_cork(&cellar_id, encoded_call.clone(), height).await {
-            Ok(res) => {
-                if res.code != 0 {
-                    error!(
-                        "failed to schedule cork for cellar {}. code {}, raw log: {}",
-                        cellar_id, res.code, res.raw_log
-                    );
-                    return Err(Status::new(
-                        Code::Internal,
-                        "cork submission failed. this may be a steward configuration problem."
-                            .to_string(),
-                    ));
-                }
-                debug!("cork response: {:?}", res)
-            }
-            Err(err) => {
-                error!("failed to schedule cork for cellar {}: {}", cellar_id, err);
-                return Err(Status::new(
-                    Code::Internal,
-                    format!("failed to send cork to sommelier: {}", err),
-                ));
-            }
+        if chain_id == ETHEREUM_CHAIN_ID {
+            handle_cork(&cellar_id, encoded_call.clone(), height).await?;
+        } else {
+            handle_axelar_cork(chain_id, &cellar_id, encoded_call.clone(), height, deadline)
+                .await?;
         }
-        let id = id_hash(height, &cellar_id, encoded_call);
+
+        let id = id_hash(height, &cellar_id, &encoded_call);
```

Comment:
I checked and currently we have the deadline in the ID hash on `axelarcork`. I think maybe since removed it from the keys we should also remove it from the ID hash, thoughts? Otherwise we'd need a different way to handle it here.

Reply:
I say keep it consistent and remove it yeah.